### PR TITLE
feat(app): implement internationalization

### PR DESCRIPTION
**Pull request by 24thsaint**

### CODE REVIEW

File: src/components/Card/OtherInformationCard.tsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
 import React from 'react'
 
+import { Profile } from '../../interfaces/Profile'
+
+import { localeInstance } from '../../i18n'
+
 import './OtherInformationCard.css'
 
-import { Profile } from '../../interfaces/Profile'
+const {
```
Comment (sianlasaga) [inline src/components/Card/OtherInformationCard.tsx]: we should move this inside the component function

File: src/components/Card/PaymentQRCard.tsx [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 import { OrdersSpend } from '../../interfaces/Order'
 import currency from '../../models/Currency'
 import Order from '../../models/Order'
-import { Button } from '../Button'
+
+import { localeInstance } from '../../i18n'
+
 import './PaymentQRCard.css'
 
+const {
```
Comment (sianlasaga) [inline src/components/Card/PaymentQRCard.tsx]: we should move this

File: src/components/Form/ListingPhotosForm.tsx [hunk 3]
```diff
@@ -4,6 +4,13 @@ import { Button } from '../Button'
 import { FormLabel } from '../Label'
 import { ThumbnavSlideshow } from '../Thumbnav'
 
+import { localeInstance } from '../../i18n'
+
+const {
```
Comment (sianlasaga) [inline src/components/Form/ListingPhotosForm.tsx]: we should move this insdie the function

File: src/components/Card/IntroductionCard.tsx [hunk 4]
```diff
@@ -2,31 +2,37 @@ import React from 'react'
...
+  const { intro } = localeInstance.get.localizations
+
+  return (
+    <div id="main-card" className="uk-card uk-card-default uk-card-body uk-width-1-2@m">
+      <div className="uk-flex uk-flex-center uk-margin-small-top">
+        <img id="introduction-logo" src="./images/Logo/Blue/SVG/Djali-Blue-Unique.svg" />
+      </div>
+      <div id="body-item">
+        <p id="description-text" className="color-primary">
+          A free market for services
```
Comment (24thsaint) [inline src/components/Card/IntroductionCard.tsx]: Translate.

File: src/components/Form/RegistrationForm.tsx [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
+import { Profile } from '../../interfaces/Profile'
 import decodeHtml from '../../utils/Unescape'
 
+import { localeInstance } from '../../i18n'
+
+import 'react-mde/lib/styles/css/react-mde-all.css'
 import './RegistrationForm.css'
 
-import { TabSelection } from '../../interfaces/Misc'
+const {
```
Comment (sianlasaga) [inline src/components/Form/RegistrationForm.tsx]: we forgot to move this inside the function

File: src/components/Header/ProfileHeader.tsx [hunk 6]
```diff
@@ -6,8 +6,17 @@ import Profile from '../../models/Profile'
 
 import config from '../../config'
 
+import { localeInstance } from '../../i18n'
+
 import './ProfileHeader.css'
 
+const {
```
Comment (sianlasaga) [inline src/components/Header/ProfileHeader.tsx]: we should move this

File: src/components/Card/OrderItemCard.tsx [hunk 7]
```diff
@@ -2,52 +2,71 @@ import React from 'react'
...
+            </div>
+          </div>
+          <span className={'order-item-title'}>{data.title}</span>
+          <span className={'mono-id order-id'}>{data.orderId}</span>
+          <span className={'label'}>{new Date(data.timestamp).toLocaleString()}</span>
+          <div className={'spacer'} />
+          <div className="uk-grid" uk-grid>
+            {source === 'sales' || source === 'cases' ? (
+              <div>
+                <div className={'label'}>Buyer</div>
```
Comment (24thsaint) [inline src/components/Card/OrderItemCard.tsx]: Translate.

File: src/components/Card/OrderItemCard.tsx [hunk 8]
```diff
@@ -2,52 +2,71 @@ import React from 'react'
...
+                <div>{data.buyerHandle}</div>
+              </div>
+            ) : null}
+            {source === 'purchases' || source === 'cases' ? (
+              <div>
+                <div className={'label'}>Vendor</div>
```
Comment (24thsaint) [inline src/components/Card/OrderItemCard.tsx]: Translate.

File: src/components/Card/OtherInformationCard.tsx [hunk 9]
```diff
@@ -25,9 +32,9 @@ const OtherInformationCard = (props: OtherInformationCardInterface) => (
         ))}
         <p id="information-title">Peer ID</p>
```
Comment (24thsaint) [inline src/components/Card/OtherInformationCard.tsx]: Translate.

File: src/components/Switcher/locale.ts [hunk 10]
```diff
@@ -0,0 +1,21 @@
+import { i18n } from '../../i18n'
```
Comment (sianlasaga) [inline src/components/Switcher/locale.ts]: remove this file

File: src/components/Modal/ModeratorInfoModal.css [hunk 11]
```diff
@@ -14,10 +14,12 @@
...
-#moderator-info-table-header > div > p {
+#moderator-info-table-header > div {
+  text-align: center;
+}
+#moderator-info-table-header > div > label {
   font-size: 0.5rem;
   color: var(--secondary);
-  text-align: center;
 }
 #moderator-info-table-body > div {
```
Comment (24thsaint) [inline src/components/Modal/ModeratorInfoModal.css]: Incorrectly formatted CSS.

File: src/components/Selector/FormSelector.tsx [hunk 12]
```diff
@@ -14,25 +15,28 @@ interface Props {
...
+      onChange={onChange}
+      required={required || false}
+      disabled={disabled}
+      value={defaultVal}
+      id={id}
+    >
+      {options ? (
+        options.map((option: Option) => (
+          <option key={option.value} value={option.value}>
+            {option.label || localizations.constants.sortOptions[option.value]}
```
Comment (24thsaint) [inline src/components/Selector/FormSelector.tsx]: Use `ID` on `sortOptions` to make component more generalized.

File: src/components/Sidebar/Filter.tsx [hunk 13]
```diff
@@ -105,36 +112,38 @@ const Filter = ({
...
-            />
+        <div className="uk-margin">
+          <FormLabel label={searchFilters.priceRangeLabel.toUpperCase()} />
+          <div className="uk-flex uk-flex-row uk-flex-center uk-flex-middle">
+            <div className="uk-inline">
+              <input
+                id={`${id}-price-min`}
+                className="uk-input"
+                type="number"
+                placeholder="MIN"
```
Comment (24thsaint) [inline src/components/Sidebar/Filter.tsx]: Translate.

File: src/components/Sidebar/Filter.tsx [hunk 14]
```diff
@@ -105,36 +112,38 @@ const Filter = ({
...
+                onChange={event => onFilterChange('priceMin', event.target.value, '<=')}
+              />
+            </div>
+            <span data-uk-icon="icon: triangle-right; ratio: 2" />
+            <div className="uk-inline">
+              <input
+                id={`${id}-price-max`}
+                className="uk-input"
+                type="number"
+                placeholder="MAX"
```
Comment (24thsaint) [inline src/components/Sidebar/Filter.tsx]: Translate.

File: src/components/Stepper/Stepper.tsx [hunk 15]
```diff
@@ -68,14 +68,14 @@ const Stepper = ({ options, currentIndex }: Props) => {
         {options.map((val, i) => {
           if (i === options.length - 1) {
             return (
-              <div key={val} className="textCont">
+              <div key={val} className="textCont uk-text-uppercase">
                 {val}
```
Comment (24thsaint) [inline src/components/Stepper/Stepper.tsx]: Translate.

File: src/components/Switcher/locale.ts [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+      en: 'Seller Ratings',
+    },
+    RedirectText: {
+      en: 'Redirecting...',
+    },
+  }
+}
+
+const locale = new Locale()
+export default locale
```
Comment (24thsaint) [inline src/components/Switcher/locale.ts]: Delete file.

**Pull request merged**